src/menu.rs

```diff
@@ -29,10 +29,18 @@ use crate::{
 };
 
 /// Object that allows you to create a `ContextMenu`.
+///
+/// ## Platform-specific
+///
+/// **Windows:** The native menu is destroyed when this stuct is dropped and can no longer be used.
```

Comment:
struct

Reply:
I forgot we removed the drop impl that destroys the native window, will remove this comment